src/main/java/io/github/hapjava/accessories/AirPurifierAccessory.java

```diff
@@ -79,8 +79,23 @@
   /** Unsubscribes from changes in the target state of the air purifier. */
   void unsubscribeTargetState();
 
+  /**
+   * If a filter maintenance service is needed as a linked service to this AirPurifier, this is the
+   * place.
+   *
+   * @return an instance of FilterMaintenanceAccessory, null if not needed.
+   */
+  default FilterMaintenanceAccessory getFilterMaintenanceAccessory() {
```

Comment:
why we need it here?
the filter service can be linked using linked Service approach. i.e. not additional method need it for it

Reply:
is the same issue as the next one as I see it.